tests/jobs/unit/aws_iot_tests_jobs_api.c

```diff
@@ -805,7 +805,7 @@ TEST( Jobs_Unit_API, SetCallback )
  */
 TEST( Jobs_Unit_API, SetCallbackMultiple )
 {
-    int32_t i = 0;
+    intptr_t i = 0;
```

Comment:
(Hygiene) Can this be changed to something more descriptive?

Reply:
That's jobs warning suppress, not related to this PR. I agree though that it should be changed for readability.